js/CutScene.js

```diff
@@ -1,37 +1,41 @@
 //cut scene to play videos between game play
 Critterer.CutScene = function(game){
      console.log('test');
-     this.Scorpions = "https://vimeo.com/112414428";
-     this.Walking_Stick = "https://vimeo.com/111256268";
-     this.Walking_Stick_Breathe = "https://vimeo.com/111256267";
-     this.Walking_Stick_do = "https://vimeo.com/111256266";
-     this.Walking_Sticks_live = "https://vimeo.com/111256265";
-     this.valueToUse;
-     this.videoend;
+      this.Scorpions = "https://vimeo.com/112414428";
+      this.Walking_Stick = "https://vimeo.com/111256268";
+      this.Walking_Stick_Breathe = "https://vimeo.com/111256267";
+      this.Walking_Stick_do = "https://vimeo.com/111256266";
+      this.Walking_Sticks_live = "https://vimeo.com/111256265";
+      this.valueToUse;
+      this.timer = game.time.create(1000, false);
```

Comment:
game.time.create() is giving an error. I haven't yet found why. But, before I dig too much further, what is this timer for? If you're trying to close the fancy box when the video is finished, have a look at [Make Vimeo Lightbox Auto-Closes on Finish Using FancyBox](http://www.sunarlim.com/2014/04/make-vimeo-lightbox-auto-closes-finish-using-fancybox/). Also, see [Vimeo's Javascript API page](https://developer.vimeo.com/player/js-api) for inspiration about what other event options are available to you.